assets/src/edit-story/components/sidebar/provider.js

```diff
@@ -24,7 +24,6 @@ import { useState, useCallback, useRef } from 'react';
 /**
  * Internal dependencies
  */
-import ColorPicker from '../../components/colorPicker';
```

Comment:
@barklund -- Changed the logic of Sidebar in this PR so that it wouldn't have to be aware of the content that's displayed in it, PTAL.